src/go/pgo/distsys/util.go

```diff
@@ -0,0 +1,17 @@
+package distsys
+
+import (
+	"math/rand"
+)
+
+const alphabet = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+/// RandString generates a random string of a specified length matching
+/// [0-9a-zA-Z]{length}.
+func RandString(length int) string {
+	b := make([]byte, length)
+	for i := range b {
+		b[i] = alphabet[rand.Intn(len(alphabet))]
+	}
+	return string(b)
+}
```

Comment:
I don't see this being used anywhere.

Reply:
It was used in the old implementation. I'm still modifying the runtime and think that this will come in handy.